### ISSUE TITLE
Something about the Equation in paper

### ISSUE THREAD

**Issue by wang149876** [hotfinda/VideoMambaPro#5]
![image](https://github.com/user-attachments/assets/ab04b40a-f8dc-414e-9669-db81bc79f617)
In the equation 29, is (i - 1) or (j - 1)?

**Comment by hotfinda** [comment]
yes you are right, it should be (i-1), we have updated the equation in our lastest version. Thanks for pointing this out!

**Comment by cx1202** [comment]
![image](https://github.com/user-attachments/assets/2b0febf3-59bb-44e6-88c8-8cfe117625be)
I don't understand why your modification of the code lead to the equation  29, as hidden states are initialized in a different way but the updation remains the same. Could you give a short explanation? Thank you!
